composer.json

```diff
@@ -31,18 +31,17 @@
         "spryker-sdk/acp": "^0.1.0",
         "spryker-sdk/async-api": "^0.2.3",
         "spryker-sdk/composer-replace": "dev-master",
-        "spryker-sdk/sdk-contracts": "dev-master",
+        "spryker-sdk/sdk-contracts": "dev-feature/sdk-635/dev-php74-compatibility",
```

Comment:
create compatibility for spryker-sdk/sdk-contracts php 7.4

Reply:
PR: https://github.com/spryker-sdk/sdk-contracts/pull/16